composer.json

```diff
@@ -12,20 +12,17 @@
 	"autoload": {
 		"files": [
 			"inc/namespace.php",
-			"inc/saml/namespace.php",
```

Comment:
@mikelittle I notice that everything `saml-related` is left and only the namespace is removed from autoload. Since I am new to these stuff I just wanted to bring it up, because it seems a bit odd.